cypress/component/DataSearch/dataset_search_filters.spec.js

```diff
@@ -20,4 +47,23 @@ describe('Data Library Filters', () => {
     cy.get('div').should('contain', 'Data Type');
     cy.get('div').should('contain', 'Participant Count');
   });
+
+  it('initially defaults to minimum and maximum participant counts in datasets', () => {
+    const props = { datasets, filters: defaultFilters(datasets), filterHandler: () => {}, isFiltered: () => {}};
+    mount(<DatasetFilterList {...props} />);
+    cy.get('#participantCountMax-range-input').should('have.value',
+      datasets[0].participantCount);
+    cy.get('#participantCountMin-range-input').should('have.value',
+      datasets[1].participantCount);
+  });
+
+  it('calls the filter handler on range changes', () => {
+    const filterHandlerStub = cy.stub();
+    const props = { datasets, filters: { ...defaultFilters(datasets), participantCountMin: 2, participantCountMax: 5 }, filterHandler: filterHandlerStub, isFiltered: () => {}};
+    mount(<DatasetFilterList {...props} />);
+    cy.get('#participantCountMax-range-input').type('3').trigger('change');
```

Comment:
Chaining calls after `type` is [unsafe](https://docs.cypress.io/api/commands/type). A lot of our tests suffer from this pattern, so it is something we should be trying to clean up.